samples/dnn/dasiamrpn_tracker.py

```diff
@@ -81,13 +103,17 @@ def __generate_anchor(self):
         score_sz = int(self.score_size)
         self.anchor = np.tile(self.anchor, score_sz * score_sz).reshape((-1, 4))
         ori = - (score_sz / 2) * self.total_stride
-        xx, yy = np.meshgrid([ori + self.total_stride * dx for dx in range(score_sz)], [ori + self.total_stride * dy for dy in range(score_sz)])
-        xx, yy = np.tile(xx.flatten(), (self.anchor_num, 1)).flatten(), np.tile(yy.flatten(), (self.anchor_num, 1)).flatten()
-        self.anchor[:, 0], self.anchor[:, 1] = xx.astype(np.float32), yy.astype(np.float32)
```

Comment:
Please do not change the lines without actual difference - it only makes review harder.

Reply:
I am sorry for the inconvenient. I thought it should be helpful to improve code style of the sample.
For now, should I reverse these changes (also, changes in the previous comment)?

Reply:
Just add a new commit which returns it to origin state. Check `git diff 3.4..dasiamrpn` to see how it looks finally.